app/routes/login.tsx

```diff
@@ -24,107 +23,72 @@ export const links: LinksFunction = () => {
-            <label>
-              <input
-                type="radio"
-                name="loginType"
-                value="login"
-                defaultChecked={
-                  !actionData?.fields?.loginType ||
-                  actionData?.fields?.loginType === "login"
-                }
-              />{" "}
-              Login
-            </label>
-            <label>
-              <input
-                type="radio"
-                name="loginType"
-                value="register"
-                defaultChecked={actionData?.fields?.loginType === "register"}
-              />{" "}
-              Register
-            </label>
-          </fieldset>
-          <div>
-            <label htmlFor="username-input">Username</label>
-            <input
-              type="text"
-              id="username-input"
-              name="username"
-              defaultValue={actionData?.fields?.username}
-              aria-invalid={fieldHasErrors(actionData, "username")}
-              aria-errormessage={
-                fieldHasErrors(actionData, "username")
-                  ? "username-error"
-                  : undefined
-              }
-            />
-            {fieldHasErrors(actionData, "username") ? (
-              <p
-                className="form-validation-error"
-                role="alert"
-                id="username-error"
-              >
-                {fieldFirstMessage(actionData, "username")}
-              </p>
-            ) : null}
-          </div>
-          <div>
-            <label htmlFor="password-input">Password</label>
-            <input
-              id="password-input"
-              name="password"
-              defaultValue={actionData?.fields?.password}
-              type="password"
-              aria-invalid={fieldHasErrors(actionData, "password")}
-              aria-errormessage={
-                fieldHasErrors(actionData, "password")
-                  ? "password-error"
-                  : undefined
-              }
-            />
-            {fieldHasErrors(actionData, "password") ? (
-              <p
-                className="form-validation-error"
-                role="alert"
-                id="password-error"
-              >
-                {fieldFirstMessage(actionData, "password")}
-              </p>
-            ) : null}
-          </div>
-          <div id="form-error-message">
-            {actionData?.errors.length ? (
-              <p className="form-validation-error" role="alert">
-                {actionData.errors.map(({ message }) => message).join(", ")}
-              </p>
-            ) : null}
-          </div>
-          <button type="submit" className="button">
-            Submit
-          </button>
+        <Form schema={registrationSchema} hiddenFields={["redirectTo"]}>
+          {({ Field, Errors, Button }) => (
+            <>
+              <Field name="loginType">
```

Comment:
I created [this issue](https://github.com/SeasonedSoftware/remix-forms/issues/9) so we can make people's lives better for similar cases in the future.

Reply:
TY for the review ... I guess we can try to clarify few concepts in remix-forms docs based on my first take ;)
An example with hidden fields would be great